compair/static/modules/gradebook/gradebook-partial.html

```diff
@@ -78,7 +78,7 @@ <h2 class="col-sm-6">
                 </td>
                 <td class="text-center" ng-if="showAttachments">
                     <span ng-if="!entry.file"> n/a </span>
-                    <a href="" ng-if="entry.file" ng-click="openAttachment(entry.file, entry.download_file_name)">View &raquo;</a>
+                    <a ng-if="entry.file" target="_blank" href="/app/attachment/{{entry.file.name}}?name={{ entry.download_file_name+'.'+entry.file.extension | encodeURIComponent}}">View &raquo;</a>
```

Comment:
It will be safer to use `ng-href` to make sure AngularJS replaced the markups before users clicking on the links.

Reply:
Fixed, thanks